actions/v2/server.go

```diff
@@ -13,6 +17,10 @@ import (
 type Server struct {
 	admin.APIAdmin
 	base.APIBase
+	data.APIData
+	users.APIUsers
+	operations.APIOperations
+	transactions.APITransactions
 }
```

Comment:
Consider this - I'm not sure if it would help us or not 🤷 
```suggestion
type Server struct {
	admin.API
	base.API
	data.API
	users.API
	operations.API
	transactions.API
}
```

Also the files could be named: `api.go`

Reply:
I thought about this in go-client. And AFAIR there is a problem that all those wrapped structs have the same name, so the go is screaming that we have duplicated field names.